src/middlewares/room.middleware.ts

```
@@ -104,3 +79,90 @@ export const updateRoomValidator = validate(
+
+    // Kiểm tra maxCapacity
+    const capacity = Number(maxCapacity)
+    if (!maxCapacity) {
+      errors.maxCapacity = {
+        type: 'field',
+        value: maxCapacity,
+        msg: 'Sức chứa tối đa là bắt buộc',
+        path: 'maxCapacity',
+        location: 'formData'
+      }
+    } else if (isNaN(capacity) || capacity < 1 || !Number.isInteger(capacity)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)